src/zoomifytilesource.js

```diff
@@ -0,0 +1,97 @@
+(function( $ ){
+
+  /**
+   * @class ZoomifyTileSource
+   * @classdesc A tilesource implementation for the zoomify format.
+   *
+   * @memberof OpenSeadragon
+   * @extends OpenSeadragon.TileSource
+   * @param {Number} width - the pixel width of the image.
+   * @param {Number} height
+   * @param {Number} tileSize
+   * @param {String} tilesUrl
+   */
+  $.ZoomifyTileSource = function( options ) {
+
+  	options.tileSize = 256;
```

Comment:
If you look at the indents here on this page, you'll see they're inconsistent; for example this line should be tabbed in more. I'm guessing you may have a mixture of tabs and spaces. Please make sure you're using only spaces with 4 spaces per indent and that your indenting is consistent.